src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -1391,4 +1544,114 @@ public function single_row( $item ) {
 	protected function get_primary_column_name() {
 		return 'name';
 	}
+
+	/**
+	 * Prints a list of other plugins that depend on the plugin.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string $dependency The dependency's filepath, relative to the plugins directory.
+	 */
+	protected function add_dependents_to_dependency_plugin_row( $dependency ) {
+		$dependent_names = WP_Plugin_Dependencies::get_dependent_names( $dependency );
+
+		if ( empty( $dependent_names ) ) {
+			return;
+		}
+
+		printf(
+			'<div class="required-by"><p><strong>%1$s</strong> %2$s</p></div>',
+			__( 'Required by:' ),
+			esc_html( implode( ' | ', $dependent_names ) )
+		);
+
+		echo '<div><p>';
+		_e( 'Note: this plugin cannot be deactivated or deleted until the plugins that require it are deactivated or deleted.' );
+		echo '</p></div>';
```

Comment:
Rather than creating a new DIV and separating the note from the "Required by" DIV, what about just adding this to the `.required-by` DIV?

```suggestion
		$dependency_note = __( 'Note: this plugin cannot be deactivated or deleted until the plugins that require it are deactivated or deleted.' );
		printf(
			'<div class="required-by"><p><strong>%1$s</strong> %2$s</p><p>%3$s</p></div>',
			__( 'Required by:' ),
			esc_html( implode( ' | ', $dependent_names ) ),
			$dependency_note
		);
```

Same suggestion for storing the note in a new `$dependent_note` variable and adding to the `sprintf()` in `::add_dependencies_to_dependent_plugin_row()`, with `<p>%3$s</p>`

Reply:
That works too, a tiny bit more complex perhaps but..?
On the other hand, perhaps the `$screen-reader_text` can be the same? The  it would just reuse the var.